migrations/helpers/mlwh_samples_update_helper.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import logging.config
+import time
+from typing import List
+import pymongo
+import sys
+import traceback
+from crawler.constants import (
+    COLLECTION_SAMPLES,
+    FIELD_CREATED_AT,
+    MONGO_DATETIME_FORMAT,
+)
+from crawler.db import (
+    create_mongo_client,
+    get_mongo_collection,
+    get_mongo_db,
+    create_mysql_connection,
+    run_mysql_executemany_query,
+)
+from crawler.helpers import (
+    get_config,
+    map_mongo_doc_to_sql_columns,
+)
+from datetime import datetime
+from crawler.sql_queries import SQL_MLWH_MULTIPLE_INSERT
+
+def valid_datetime_string(s_datetime: str) -> bool:
+    try:
+        dt = datetime.strptime(s_datetime, MONGO_DATETIME_FORMAT)
+        if dt == None:
+            return False
+        return True
+    except Exception as e:
+        print_exception()
+        return False
+
+def print_exception() -> None:
+    print(f'An exception occurred, at {datetime.now()}')
+    e = sys.exc_info()
+    print(e[0]) # exception type
+    print(e[1]) # exception message
+    if e[2]: # traceback
+      traceback.print_tb(e[2], limit=10)
```

Comment:
like I said somewhere else, could replace with logger.exception(e)

Reply:
I was using print here because I copied the samples migration. This migration is run at the command line, so maybe logger.exception is not ideal as not visible?

Reply:
I wasn't sure if logger messages would be visible or not - have you tested it? Of course, if they are not visible then stick with print!

Reply:
Tried it and they don't show.

Reply:
:(